lib/epics/client.rb

```diff
@@ -159,34 +167,67 @@ def HTD
     end.to_xml
   end
 
+  # fetch bank parameters
   def HPD
     download(Epics::HPD)
   end
 
+  # fetch client and subscriber data
   def HKD
     download(Epics::HKD)
   end
 
+  # fetch text protocol
   def PTK(from, to)
     download(Epics::PTK, from, to)
   end
 
+  # XML Protokolldatei abholen
   def HAC(from = nil, to = nil)
     download(Epics::HAC, from, to)
   end
 
+  # VEU related actions
+
+  # Fetch overview of all orders which need to be signed
+  def HVU
+    Epics::H004.from_xml(download(Epics::HVU)).to_h
```

Comment:
do we really need this intermediate class `Epics::H004` which determines the type and reacts accordingly? I mean in this method we already know that the result will be `HVU` anyways.